src/shared/styles/components/add-contact-modal.css

```
@@ -4,10 +4,12 @@
   background-color: rgb(63, 63, 63);
   border: 2px solid #444;
   border-radius: 12px;
-  padding: 1rem;
-  width: clamp(220px, 80vw, 600px);
+  padding: 1rem 1.5rem;
   color: white;
   position: relative;
+
+  width: clamp(220px, 100vw, 440px);
```

Stylelint (17.6.0)

[error] Unexpected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)